internal/kube/rootlessdynamic/client.go

```diff
@@ -57,18 +56,18 @@ func (c *client) Apply(ctx context.Context, resource *unstructured.Unstructured)
 		// we should not expect here for all resources to be installed in the kyma-system namespace. passed resources should be defaulted and validated out of the Apply func
 		err = c.applyFunc(ctx, c.dynamic.Resource(*gvr).Namespace("kyma-system"), resource)
 		if err != nil {
-			return clierror.Wrap(err, clierror.New("failed to apply namespaced resource"))
+			return err
 		}
 	} else {
 		err = c.applyFunc(ctx, c.dynamic.Resource(*gvr), resource)
 		if err != nil {
-			return clierror.Wrap(err, clierror.New("failed to apply cluster-scoped resource"))
+			return err
```

Comment:
I would suggest wrapping error from the `c.applyFunc` with the `failed to apply cluster-scoped resource` message as it was before. The same goes for the rest of the returns